src/blocks/timewarrior.rs

```diff
@@ -0,0 +1,223 @@
+//! Time and information of the current timewarrior task
+//!
+//! Clicking left mouse stops or resumes the task
+//!
+//! # Configuration
+//!
+//! Key | Values | Default
+//! ----|--------|--------
+//! `interval` | Update interval in seconds | `30`
+//! `format` | A string to customise the output of the block. See placeholders. | `" $icon {$elapsed\|}"`
+//! `info` | The threshold of minutes the task turns into a info state | -
+//! `good` | The threshold of minutes the task turns into a good state | -
+//! `warning` | The threshold of minutes the task turns into a warning state | -
+//! `critical` | The threshold of minutes the task turns into a critical state | -
+//!
+//! Placeholder | Value | Type | Unit
+//! ------------|-------|------|------
+//! `icon`   | A static icon | Icon | -
+//! `elapsed`| Elapsed time in format H:MM (Only present if task is active) | Text | -
+//! `tags`   | Tags of the active task separated by space (Only present if task is active) | Text | -
+//! `annotation` | Annotation of the active task (Only present if task is active) | Text | -
+//!
+//! # Example
+//! ```toml
+//! [[block]]
+//! block  = "timewarrior"
+//! format = " $icon {$tags.rot-str(8,1) $elapsed|}"
+//! ```
+//!
+//! # Icons Used
+//! - `tasks`
+
+use super::prelude::*;
+use chrono::DateTime;
+use tokio::process::Command;
+
+#[derive(Deserialize, Debug, SmartDefault)]
+#[serde(default)]
+pub struct Config {
+    #[default(30.into())]
+    interval: Seconds,
+    format: FormatConfig,
+
+    info: Option<u64>,
+    good: Option<u64>,
+    warning: Option<u64>,
+    critical: Option<u64>,
+}
+
+pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
+    api.set_default_actions(&[(MouseButton::Left, None, "stop_continue")])
```

Comment:
Please document the action name and that it is the default in a table in the docs (see music for example).